tests/queries/0_stateless/02205_HTTP_user_agent.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+#. "$CURDIR"/../shell_config.sh
```

Comment:
Why do you comment this line? It is used for tests (run script) and probably it is the reason for failure

Reply:
I think I may have commented this line out when I was trying to figure out what the script was doing, thank you.

Reply:
The CI pipeline is showing a `500` internal server error which I encountered during the development of the test when the query was malformed.

Reply:
I see the issue, I should have been using the `HTTP_SERVER_URL_STR`